<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_finalizar_venda</class>
 <widget class="QDialog" name="tela_finalizar_venda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource_file.qrc">
    <normaloff>:/logo cin.png</normaloff>:/logo cin.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(230, 230, 230);
color:black;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="btn_cancelar">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
	background-color:rgb(255, 150, 150);
    border: 1px solid black;
    border-radius: 5px;
    padding: 2px 10px;
}

QPushButton:hover {
    background-color: rgb(255, 131, 131);
}

QPushButton:pressed {
    background-color: rgb(255, 131, 131);
}
</string>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Total:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_total">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Desconto% :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="singleStep">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Total:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_total_desc">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Forma de Pagamento:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(255, 255, 255);
color: black</string>
         </property>
         <item>
          <property name="text">
           <string>Dinheiro</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Credito</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Debito</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pix</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btn_fechar_venda">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
	background-color:rgb(255, 150, 150);
    border: 1px solid black;
    border-radius: 5px;
    padding: 2px 10px;
}

QPushButton:hover {
    background-color: rgb(255, 131, 131);
}

QPushButton:pressed {
    background-color: rgb(255, 131, 131);
}
</string>
     </property>
     <property name="text">
      <string>Fechar Venda</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource_file.qrc"/>
 </resources>
 <connections/>
</ui>
